tests/upgrades/test_repository.py

```diff
@@ -33,75 +33,6 @@
 )
 
 
-class Scenario_repository_upstream_authorization_check(APITestCase):
-    """ This test scenario is to verify the upstream username in post-upgrade for a custom
-    repository which does have a upstream username but not password set on it in pre-upgrade.
-
-    Test Steps:
-
-        1. Before Satellite upgrade, Create a custom repository and sync it.
-        2. Set the upstream username on same repository using foreman-rake.
-        3. Upgrade Satellite.
-        4. Check if the upstream username value is removed for same repository.
-    """
-
-    @classmethod
-    def setUpClass(cls):
-        cls.upstream_username = 'rTtest123'
-
-    @pre_upgrade
-    def test_pre_repository_scenario_upstream_authorization(self):
-        """ Create a custom repository and set the upstream username on it.
-
-        :id: preupgrade-11c5ceee-bfe0-4ce9-8f7b-67a835baf522
-
-        :steps:
-            1. Create a custom repository and sync it.
-            2. Set the upstream username on same repository using foreman-rake.
-
-        :expectedresults:
-            1. Upstream username should be set on repository.
-
-        :BZ: 1641785
-        """
-
-        org = entities.Organization().create()
-        custom_repo = create_sync_custom_repo(org_id=org.id)
-        rake_repo = 'repo = Katello::Repository.find_by_id({0})'.format(custom_repo)
```

Comment:
@vijay8451 I see upstream username and password in UI. May be rake command changed here?

Reply:
@ntkathole yeah it's there but you can not update the standalone username or password as you have to give both in same time.
Another way do earlier in 6.4 that was `rake console` but check is made from `6.5` hence it is  not allowing now same and applicable too:
```
irb(main):002:0> repo.upstream_
repo.upstream_password  repo.upstream_username  
irb(main):002:0> repo.upstream_username = 'test123'
Traceback (most recent call last):
        2: from lib/tasks/console.rake:5:in `block in <top (required)>'
        1: from (irb):2
NoMethodError (undefined method `upstream_username=' for #<Katello::Repository:0x000000000ad6cfb8>)
Did you mean?  upstream_username
irb(main):003:0> repo.upstream_password = 'password'
Traceback (most recent call last):
        2: from lib/tasks/console.rake:5:in `block in <top (required)>'
        1: from (irb):3
NoMethodError (undefined method `upstream_password=' for #<Katello::Repository:0x000000000ad6cfb8>)
Did you mean?  upstream_password
irb(main):004:0> 

```

Reply:
@vijay8451 Could you please try with `repo.root.upstream_username = "test123"` ? This is changed via https://projects.theforeman.org/issues/24328

Reply:
@ntkathole great to see that .. I was not aware about it. let me just update the command only that all :)